### PR TITLE
Fix status and date on waiting list

### PR DESCRIPTION
**Pull request by steveklabnik**
Make both of these fields work.

There's a small amount of DRY violation going on, and it's possible that instead of computed properties, these should be helpers. I slightly prefer computed properties but it's no biggie either way.

### CODE REVIEW

File: app/models/customer.js [hunk 1]
```diff
@@ -12,4 +12,22 @@ export default DS.Model.extend({
   credit: DS.attr('string'),
   want: DS.attr('string'),
   notes: DS.attr('string'),
+
+  statusText: function() {
```
Comment (locks) [inline app/models/customer.js]: nitpick: `Ember.computed('status', function() {` is currently preferred to prototype extension syntax

Comment (steveklabnik) [inline app/models/customer.js]: 👍❤️

**Comment by ashleygwilliams** [comment]
r+, (though i sort think these should be helpers, fine for now)

**Comment by ashleygwilliams** [comment]
squash :eggplant: plz :wink:

**Comment by steveklabnik** [comment]
Squashed!

**Pull request merged**